.github/10-documentation-issue.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Documentation Issue
+about: Use this template for documentation related issues
+
+---
+
+<em>tag:doc_template</em>
+
+**Please provide the following information**
+
+**System information**
+- TensorFlow version :
+- Tensorboard version :
```

Comment:
nit: first word should be capitalized `TensorBoard` with a capital B

Reply:
Done

Reply:
I think you are missing a commit